static/js/github-setup.js

```diff
@@ -18,6 +18,29 @@ const hasProtocol = (str) =>
 	str.includes("http://") || str.includes("https://") || str.includes("www.");
 
 $(document).ready(() => {
+	// Handle form submit
+	$(".githubSetup__form").on("submit", (event) => {
+		event.preventDefault();
+		// TODO do something here to show user that form is being submitted
+		$.post(
+			event.target.action,
+			Object.fromEntries(new FormData(event.target).entries())
+		)
+			.done(function(body) {
+				if (body.error) {
+					throw body;
+				}
+
+				// Redirecting the user to the correct jira instance
+				// Should probably redo this with query strings and just redoing the GET request
+				document.location.href = body.redirect;
+			})
+			.fail(function(response) {
+				// Handle failure
```

Comment:
What's the plan here?

Reply:
Dunno.  It was never able to deal with an error in the first place. I created a ticket for assessing the github/setup flow.